README.md

```diff
@@ -58,13 +53,39 @@ You must have a docker container built corresponding to the submodule below or t
 Cd into the AutoGPT submodule and build/tag the dockerfile so the agent can be instantiated.
 `cd auto_gpt_benchmarks/Auto-GPT`
 
-Build the container so we can run it procedurally!
+Build the container so we can run it procedurally! Make sure to have docker running on your computer.
 `docker build -t autogpt .`
 
+After docker is built, navigate to Auto-GPT\Auto-GPT-Benchmarks directory and set the API Key in command line for windows/mac/linux below:
+
+(Windows) set OPENAI_API_KEY=<insert here>
```

Comment:
This looks good. Can you just update it and make sure it doesn't conflict with the minor readme changes I just merged in from your partner in crime?

Reply:
yea sure